feature-libs/organization/administration/components/user/change-password-form/user-change-password-form.service.ts

```diff
@@ -4,34 +4,27 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Injectable, inject } from '@angular/core';
+import { Injectable } from '@angular/core';
 import {
   UntypedFormControl,
   UntypedFormGroup,
   Validators,
 } from '@angular/forms';
-import { FeatureConfigService, User } from '@spartacus/core';
+import { User } from '@spartacus/core';
 import { CustomFormValidators } from '@spartacus/storefront';
 import { FormService } from '../../shared/form/form.service';
 
 @Injectable({
   providedIn: 'root',
 })
 export class UserChangePasswordFormService extends FormService<any> {
-  private featureConfigService = inject(FeatureConfigService);
-
   /**
    * @override
    * Adds the password and confirmPassword field. Also adds the customerId field,
    * so that the customerId can be used during persistent.
    */
   protected build() {
-    // TODO: (CXSPA-7315) Remove feature toggle in the next major
-    const passwordValidators = this.featureConfigService?.isEnabled(
-      'formErrorsDescriptiveMessages'
-    )
-      ? [CustomFormValidators.passwordValidator]
-      : CustomFormValidators.passwordValidators;
+    const passwordValidators = CustomFormValidators.passwordValidators;
```

Comment:
Should not it be rather 
```ts
const passwordValidators = [CustomFormValidators.passwordValidator];
```

Reply:
Ah ok, looks like you corrected it, but previously it was handled wrongly, right?